backend/dataall/db/permissions.py

```diff
@@ -316,7 +295,6 @@
 RESOURCES_ALL_WITH_DESC = {k: k for k in RESOURCES_ALL}
 RESOURCES_ALL_WITH_DESC[CREATE_DASHBOARD] = 'Create dashboards on this environment'
 RESOURCES_ALL_WITH_DESC[CREATE_REDSHIFT_CLUSTER] = 'Create Redshift clusters on this environment'
-RESOURCES_ALL_WITH_DESC[CREATE_SGMSTUDIO_NOTEBOOK] = 'Create ML Studio profiles on this environment'
```

Comment:
We should not forget about `MANAGE_SGMSTUDIO_NOTEBOOKS`

Reply:
Yes, I noticed while testing :S